tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, test_context):
     def setup_services(self, security_protocol=SecurityConfig.PLAINTEXT, timestamp_type=None, broker_version=DEV_BRANCH, auto_create_topics=False):
         self.kafka = KafkaService(self.test_context, self.num_brokers, self.zk,
                                   security_protocol=security_protocol, interbroker_security_protocol=security_protocol,
-                                  topics=self.topics, version=broker_version,
+                                  topics=self.topics, version=KafkaVersion(broker_version),
```

Comment:
I think it would be better to update callers of `setup_services` to use KafkaVersion objects since at the moment they are converting to string. The default value `DEV_BRANCH` used here is already a KafkaVersion instance. Looks like we only need to update the parameters in this method: https://github.com/apache/kafka/blob/da07985b0166c7048b7e92d54fbc33bea1d31da5/tests/kafkatest/tests/connect/connect_distributed_test.py#L530.